docs/integrations/notifications/one-signal/getting-started.md

```diff
@@ -104,4 +104,4 @@ You can find out if the user was successfuly added to the subscription by naviga
 ![img_2.png](img_2.png)
 
 
-
+Note: Currently, OneSignal integration supports Firebase authentication. If you want to use Supabase authentication, you will need to provide custom code to notify your users.
```

Comment:
hey @idea-garage, for notes like this, we should use [admonitions](https://docusaurus.io/docs/markdown-features/admonitions). Probably a warning admonition with a title "One Signal for Supabase Users" may be good! Here is our [contribution guidelines for more info ](https://flutterflow.notion.site/Docs-Contribution-Guidelines-42829c305df441f6b061b4095abac8a8#477a2547e65642efb4b8f83ba671ddc9)

Reply:
Thanks. I tried to use Admonitions and sent another PR. It looks Admonitions doesn't shown in preview.

Reply:
What are you using for previews? 
Can you also add a title to your Admonitions? Like: 

:::info[OneSignal for Supabase Users]
<text>
::: 


Also please resolve conflicts if you can. Otherwise I can add the change directly on my end.

Reply:
Sorry, I couldn't resolve the conflict on my end. Please take care of it?

Reply:
Yes, I added the changes directly. Sorry, a lot of things are being pushed from our end, so i just added the one liner change directly. Closing this issue for now.